### DISCUSSION TITLE
What does "Add parents folder as tag?" mean?

### DISCUSSION THREAD

**Discussion by l0b0** [go-shiori/shiori#750] [Q&A] [answered]
This question comes up when importing a HTML file using the CLI, and I've no idea what it means.

**Comment by fmartingr** [comment, marked as answer]
Basically the Netscape export format have a list of URLs which are the bookmarks, but hey can be grouped around like lists are. This question is to add those group names as tags as well.

Something like this:

```
- group1
  - url1
- group2
  - url2
```

In this case, `group1` would be added as bookmark tag for `url1` (and `group2` for `url2`). It depends on the export file.
